packages/eds-core-react/src/components/Popover/Popover.docs.mdx

```diff
@@ -0,0 +1,113 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Popover } from '.'
+
+# Popover
+
+Is a floating card that provides more information or actions on hover or click.
+
+<Links 
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=17515%3A0"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/8809df-popover/b/398446"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Popover/Popover.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="data-display-popover--introduction"/>
+<PropsTable />
+
+## Usage
+
+`Popover` provides excess or additional information that does not fit in the main content area. 
+<ul>
+    <li>A popover displays rich content in a non-modal dialog to describe or add additional
+        information when users hover over, focus on, or click an interactive element.</li>
+    <li>User can interact with popover content.</li>
+    <li>Use sparingly.</li>
+  </ul>
+
+  ```tsx
+import { Popover } from '@equinor/eds-core-react'
+
+const Demo = () => {
+    const [isOpen, setIsOpen] = useState(false)
+    const handleOpen = () => {
+      setIsOpen(true)
+    }
+    const handleClose = () => {
+      setIsOpen(false)
+    }
+
+    return (
+        <>
+      <Button
+        aria-haspopup
+        id="default-popover-anchor"
+        aria-controls="default-popover"
+        aria-expanded={isOpen}
+        ref={referenceElement}
```

Comment:
I guess this is not important but the referenceElement is never set in this example code